src/app/ReadHandler.cpp

```diff
@@ -919,5 +919,17 @@ void ReadHandler::ClearStateFlag(ReadHandlerFlags aFlag)
     SetStateFlag(aFlag, false);
 }
 
+size_t ReadHandler::GetReportBufferMaxSize()
+{
+    size_t maxBufSize                  = chip::app::kMaxSecureSduLengthBytes;
```

Comment:
This is all inside chip::app, no?
```suggestion
    size_t maxBufSize                  = kMaxSecureSduLengthBytes;
```